src/index.ts

```diff
@@ -40,21 +50,39 @@ export function getServeCommand<RawConfiguration, Configuration, State>(
     .addOption(
       new Option("--port <port>")
         .env("PORT")
-        .default(8100)
+        .default(8080)
         .argParser(parseIntOption)
     )
     .addOption(
       new Option("--service-token-secret <secret>").env("SERVICE_TOKEN_SECRET")
     )
-    .addOption(new Option("--otlp_endpoint <endpoint>").env("OTLP_ENDPOINT"))
-    .addOption(new Option("--service-name <name>").env("OTEL_SERVICE_NAME"))
-    .addOption(new Option("--log-level <level>").env("LOG_LEVEL").default("info"))
-    .addOption(new Option("--pretty-print-logs").env("PRETTY_PRINT_LOGS").default(false));
+    .addOption(
+      new Option("--otel-exporter-otlp-endpoint <endpoint>")
+        .env("OTEL_EXPORTER_OTLP_ENDPOINT")
+        .default("http://localhost:4318")
+    )
+    .addOption(
+      new Option("--otel-service-name <name>")
+        .env("OTEL_SERVICE_NAME")
+        .default("hasura-ndc")
+    )
+    .addOption(
+      new Option("--log-level <level>").env("LOG_LEVEL").default("info")
+    )
+    .addOption(
+      new Option("--pretty-print-logs").env("PRETTY_PRINT_LOGS").default(false)
+    );
 
   if (connector) {
     command.action(async (options: ServerOptions) => {
+      if (options.otelExporterOtlpEndpoint) {
+        initTelemetry(
+          options.otelServiceName,
+          options.otelExporterOtlpEndpoint
+        );
```

Comment:
Is there a reason this wasn't added inside `startServer`? 

For users of the SDK [that provide their own server startup action](https://github.com/hasura/ndc-nodejs-lambda/blob/main/ndc-lambda-sdk/src/cmdline.ts#L19-L23) (that [eventually calls `startServer`](https://github.com/hasura/ndc-nodejs-lambda/blob/main/ndc-lambda-sdk/src/host.ts#L6)), it would be nice if this was just done for them inside `startServer` so they don't need to do it themselves.

Reply:
You're right, this should go inside startServer